setup-scripts/reset-tests.sh

```diff
@@ -2,5 +2,5 @@
 
 bash scripts/teardown-tests.sh
 
-bash scripts/setup-tests.sh
+bash setup-scripts/setup-tests.sh
```

Comment:
I don't think we need both `scripts/` and `setup-scripts/`, maybe put the setup ones inside a subdir of `scripts`? e.g.:

```
scripts/
scripts/<remaining shell scripts>
scripts/setup_scripts/
scripts/setup_scripts/<moved scripts>
```

Reply:
Defer to later as I prefer users to not have to interact with scripts under `/scripts`